.pre-commit-config.yaml

```
@@ -54,18 +57,12 @@ repos:
       rev: v1.12.1
       hooks:
           - id: blacken-docs
-            additional_dependencies: [black~=22.0]
+            additional_dependencies:
+            - black==23.10.1
```

```suggestion
            - black~=23.0
```
3.7 compatibility...